templates/practice_settings/general_list.php

```diff
@@ -1,7 +1,7 @@
 <html>
 <head>
 <link rel="stylesheet" href="<?php echo $this->css_header;?>" type="text/css">
-<?php call_required_libraries(true,true,false,false);?>
+<?php call_required_libraries(bootstrap=true,fancybox=true,knockout=false,datepicker=false); ?>
```

Comment:
I guess we are still using the /templates directory?

Reply:
ok,  I'm looking into it @aethelwulffe

Reply:
Yes, but the smarty templates are removed.